client/src/components/Form/Elements/FormData/FormExampleData.vue

```diff
@@ -0,0 +1,28 @@
+<script setup lang="ts">
+import type { ExampleData } from "./types";
+
+import FormExampleDataElement from "./FormExampleDataElement.vue";
+
+const props = defineProps<{
+    value: ExampleData;
+}>();
+</script>
+
+<template>
+    <FormExampleDataElement
+        v-if="props.value.class === 'Collection'"
```

Comment:
All types of parameters can be specified in a landing request. While you're at it, please don't call it example data, that's not what this is.

Reply:
Yes, this is a really rough draft (especially the types I've added). I was trying to look locally for a schema of some sort for all examples of landing data.

Will polish this a lot more once I get back to this